src/test/App.test.js

```diff
@@ -0,0 +1,38 @@
+// import { render, screen } from '@testing-library/react';
+// import { BrowserRouter } from 'react-router-dom';
+// import { Provider } from 'react-redux';
+// import store from '../redux/store/store';
+// import App from '../App';
+
+// test('It should Should display the header', () => {
+//   render(
+//     <Provider store={store}>
+//       <BrowserRouter>
+//         <App />
+//       </BrowserRouter>
+//     </Provider>,
+//   );
+//   const headerText = screen.getByText(/COVID-19 GLOBAL STAT/i);
+//   expect(headerText).toEqual(headerText);
+// });
```

Comment:
- [ ] Please kindly remove commented code from your code base as it makes your code dirty and less readable.

Reply:
Alright, done